### PR TITLE
Note

### PR DESCRIPTION
**Pull request by andymunro**
I moved the note to be after the metadata and changed the asciidoc coding for this note.

### CODE REVIEW

**Comment by stianst** [comment]
Is there a JIRA for this work?

**Review by iankko — CHANGES_REQUESTED** [review]
@andymunro Hey Andy,

  wondering what's the purpose of this one (since it contains commit from https://github.com/keycloak/keycloak-documentation/pull/759 one too). 

Was the intention to fix just [KEYCLOAK-11783](https://issues.jboss.org/browse/KEYCLOAK-11783) within this one? If that's the case, can you remove the first commit related with other, unrelated ([KEYCLOAK-7010](https://issues.jboss.org/browse/KEYCLOAK-7010)) JIRA?

Thanks, Jan

**Comment by andymunro** [comment]
Yes, thanks, @iankko,  that was not intentional.  I removed it.

File: server_installation/topics/operating-mode/crossdc.adoc [hunk 1]
```diff
@@ -2,12 +2,15 @@
...
 === Cross-Datacenter Replication Mode
 
-NOTE: Cross-Datacenter Replication Mode is Technology Preview and is not fully supported.
-
 :tech_feature_name: Cross-Datacenter Replication Mode
 :tech_feature_disabled: false
 include::../templates/techpreview.adoc[]
 
+[NOTE]
+====
```
Comment (stianst) [inline server_installation/topics/operating-mode/crossdc.adoc:10]: This shouldn't be needed as it's already in techpreview.adoc. Looks like for some reason that is not working.

**Comment by stianst** [comment]
Closing this - proper PR here: https://github.com/keycloak/keycloak-documentation/pull/775

**Pull request closed without merging**